integration-test/src/main/java/com/sequenceiq/it/cloudbreak/actor/CloudbreakUserCache.java

```diff
@@ -89,30 +92,49 @@ public CloudbreakUser getByDisplayName(String name) {
         if (MapUtils.isEmpty(usersByAccount)) {
             initRealUmsUserCache();
         }
-        CloudbreakUser user = usersByAccount.values().stream().flatMap(Collection::stream)
-                .filter(u -> u.getDisplayName().equals(name)).findFirst()
-                .orElseThrow(() -> new TestFailException(String.format("There is no real UMS user with::%n name: %s%n deployment::account: %s::%s", name,
-                        realUmsUserDeployment, realUmsUserAccount)));
-        LOGGER.info(" Real UMS user has been found:: \nDisplay name: {} \nCrn: {} \nAccess key: {} \nSecret key: {} \nAdmin: {} ",
-                user.getDisplayName(), user.getCrn(), user.getAccessKey(), user.getSecretKey(), user.getAdmin());
-        return user;
+        if (MapUtils.isNotEmpty(usersByAccount)) {
+            CloudbreakUser user = usersByAccount.values().stream().flatMap(Collection::stream)
+                    .filter(u -> u.getDisplayName().equals(name)).findFirst()
+                    .orElseThrow(() -> new TestFailException(String.format("There is no real UMS user with::%n name: %s%n deployment::account: %s::%s", name,
+                            realUmsUserDeployment, realUmsUserAccount)));
+            LOGGER.info(" Real UMS user has been found:: \nDisplay name: {} \nCrn: {} \nAccess key: {} \nSecret key: {} \nAdmin: {} ",
+                    user.getDisplayName(), user.getCrn(), user.getAccessKey(), user.getSecretKey(), user.getAdmin());
+            return user;
+        } else {
+            throw new TestFailException("UMS user store 'ums-users/api-credentials.json' is not available." +
+                    " So initialization of real UMS user cache is not possible!");
+        }
     }
 
     public CloudbreakUser getAdminByAccountId(String accountId) {
-        LOGGER.info("Getting the requested real UMS admin by account Id: {}", accountId);
-        try {
-            CloudbreakUser adminUser = usersByAccount.values().stream().flatMap(Collection::stream)
-                    .filter(CloudbreakUser::getAdmin).findFirst()
-                    .orElseThrow(() -> new TestFailException(String.format("There is no real UMS admin in account: %s", accountId)));
-            LOGGER.info(" Real UMS account admin has been found:: \nDisplay name: {} \nCrn: {} \nAccess key: {} \nSecret key: {} \nAdmin: {} ",
-                    adminUser.getDisplayName(), adminUser.getCrn(), adminUser.getAccessKey(), adminUser.getSecretKey(), adminUser.getAdmin());
-            return adminUser;
-        } catch (Exception e) {
-            throw new TestFailException(String.format("Cannot get the real UMS admin in account: %s, because of: %s", accountId, e.getMessage()), e);
+        if (!usersByAccount.containsKey(accountId)) {
+            throw new TestFailException("Account mismatch:: Real UMS '" + accountId + "' account ID is not present! Please make sure: " +
```

Comment:
I would simplify this message, like this: `accountId .. is missing from ums-users/api-credential.json`